core/compilers.ts

```diff
@@ -37,10 +37,7 @@ export function compile(code: string, path: string): string {
   }
   if (Path.fileExtension(path) === "sql") {
     const { sql } = extractSqlxParts(SyntaxTreeNode.create(code));
```

Comment:
i think this is pretty heavyweight, just for the sake of some escaping. i think you should just

```
const escapedCode = code.replace('`', '\\`')}.replace('${', '\\${');
return `exports.query = `${escapedCode}`;`;
```

Reply:
Wait, why `.replace('${', '\\${')`? I thought the chat meant no JS in the SQL at all, not just no contextables